packages/docs/Next/Content/BlockQuote.md

```
@@ -0,0 +1,29 @@
+# Block Quote
+
+### Default
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)